save-backend/src/main/kotlin/com/saveourtool/save/backend/controllers/DemoManagerController.kt

```diff
@@ -333,4 +333,44 @@ class DemoManagerController(
                     )
                 )
         }
+
+    @PostMapping("/{organizationName}/{projectName}/delete")
+    @RequiresAuthorizationSourceHeader
+    @PreAuthorize("hasRole('ROLE_SUPER_ADMIN')")
```

Comment:
do we still need it taking into account that you check permission on project level?